queries.sql

```diff
@@ -77,3 +77,45 @@ SELECT species, AVG(escape_attempts) AS avg_escape_attempts
 FROM animals
 WHERE YEAR(date_of_birth) BETWEEN 1990 AND 2000
 GROUP BY species;
+
+/* Day 3 of query multiple tables*/
+
+SELECT a.name
+FROM animals AS a
+JOIN owners AS o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
```

Comment:
👍 Great work on your query! However, there's one important step remaining:

- [ ] As per the project requirement, please update the screenshot for this query. Your screenshot should reflect this query, showing the results. 


By updating the screenshot to align with the project requirements, your documentation will be complete and accurate. Keep up the great work! 👍📊🔄

Reply:
- [ ] @richie1988 Great work implementing the changes requested, however, the change should be implemented in the pull request description, kindly consider updating the screenshot as previously requested.
![image](https://github.com/richie1988/vet_clinic/assets/61408860/fb2b26e3-7d2a-4a8f-9fc2-15bcdd062fda)